app/Console/Kernel.php

```diff
@@ -36,6 +37,7 @@ protected function schedule(Schedule $schedule)
         $schedule->job(new ExpireOldUserVerificationTokensJob)->hourly();
         $schedule->job(new PruneEventPageUpdatesTable)->everyFifteenMinutes();
         $schedule->job(new PruneQueryserviceBatchesTable)->everyFifteenMinutes();
+        $schedule->job(new CreateQueryserviceBatchesJob)->everyMinute();
```

Comment:
This introduces reasonable lag to the system. Not that I think we shouldn't do this but we should communicate this to Eve and Valerie to tell the users. They should *know* that the mean time to hit the queryservice is like 35 seconds (30s here and 5s in the updater polling)

Reply:
yeah, it's totally fine IMHO